File: modules/auth/templates/backend/express/src/auth/crypto.js
/**
 * Password / token helper placeholders for {{PACKAGE_NAME}}.
 * Installs jsonwebtoken + bcryptjs via module.json — implement later.
 */
export async function hashPassword(_password) {
  throw new Error("hashPassword not implemented");
}

export async function verifyPassword(_password, _hash) {
  throw new Error("verifyPassword not implemented");
}

export function signToken(_payload) {
  throw new Error("signToken not implemented");
}

export function verifyToken(_token) {
  throw new Error("verifyToken not implemented");
}

File: modules/auth/templates/backend/fastapi/auth/crypto.py
"""Password / token helper placeholders for {{PACKAGE_NAME}}."""


def hash_password(_password: str) -> str:
    raise NotImplementedError("hash_password not implemented")


def verify_password(_password: str, _hash: str) -> bool:
    raise NotImplementedError("verify_password not implemented")


def sign_token(_payload: dict) -> str:
    raise NotImplementedError("sign_token not implemented")


def verify_token(_token: str) -> dict:
    raise NotImplementedError("verify_token not implemented")
